src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelectorCore.js

```diff
@@ -35,11 +35,16 @@ type MentionStartStateProps = {
     message?: React.Node,
 };
 
-const MentionStartState = ({ message }: MentionStartStateProps) => <div className="mention-start-state">{message}</div>;
+const MentionStartState = ({ message }: MentionStartStateProps) => (
+    <div role="alert" className="mention-start-state">
+        {message}
+    </div>
```

Comment:
Would it make sense to have a single alert component that just takes a message that could be used for mention start as well as when there are active mentions listed?

Reply:
because we are running out of time... I guess a refactor for the other messages and components can be worked in a different ticket